src/third_party/html-parser/html-parser.ts

```diff
@@ -232,3 +232,16 @@ export function parse(data: string, rootElement: Element) {
 
   throw new Error('Attempting to parse invalid HTML.');
 }
+
+function decodeNumericEntities(html: string) {
+  return html.replace(/&#(x?[\da-fA-F]+);?/g, function (s, entity) {
```

Comment:
The `x` can be `X`, too.

```suggestion
  return html.replace(/&#(x?[\da-f]+);?/gi, function (s, entity) {
```